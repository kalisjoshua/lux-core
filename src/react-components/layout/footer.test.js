import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './footer';

describe('Default layout - footer', function () {
  it('should match the snapshot', function () {
    const component = renderer.create(
      <Footer />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
