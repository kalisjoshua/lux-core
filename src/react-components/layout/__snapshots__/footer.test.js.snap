exports[`Default layout - footer should match the snapshot 1`] = `
<footer
  role="contentinfo">
  <p>
    Â© 
    2016
     
    <a
      className="link--plain"
      href="http://quickenloans.com">
      Quicken Loans
    </a>
  </p>
   
</footer>
`;
